python/pytests/aggregation/count_if_test.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import kaskada as kd
+
+
+@pytest.fixture
```

Comment:
Since this is used multiple times, there would be a benefit to using `@pytest.fixture(scope = "module")`. Specifically, that would share the source across all test in this file (module) rather than creating (and preparing / loading) a new source for each test.